src/commands/misc_commands/news.ts

```diff
@@ -0,0 +1,443 @@
+                    description: i18n.translate(
+                        LocaleType.EN,
+                        "command.news.help.interaction.unsubscribe",
+                    ),
+                    description_localizations: Object.values(LocaleType)
+                        .filter((x) => x !== LocaleType.EN)
+                        .reduce(
+                            (acc, locale) => ({
+                                ...acc,
+                                [locale]: i18n.translate(
+                                    locale,
+                                    "command.news.help.interaction.unsubscribe",
+                                ),
+                            }),
+                            {},
+                        ),
+                    type: Eris.Constants.ApplicationCommandOptionTypes
+                        .SUB_COMMAND,
+                    options: [
+                        {
+                            name: RANGE_OPTION,
+                            description: i18n.translate(
+                                LocaleType.EN,
+                                "command.news.help.interaction.range",
+                            ),
+                            description_localizations: Object.values(LocaleType)
+                                .filter((x) => x !== LocaleType.EN)
+                                .reduce(
+                                    (acc, locale) => ({
+                                        ...acc,
+                                        [locale]: i18n.translate(
+                                            locale,
+                                            "command.news.help.interaction.range",
+                                        ),
+                                    }),
+                                    {},
+                                ),
+                            type: Eris.Constants.ApplicationCommandOptionTypes
+                                .STRING,
+                            required: true,
+                            choices: Object.values(NewsRange).map(
+                                (newsRange) => ({
+                                    name: newsRange,
+                                    value: newsRange,
+                                }),
+                            ),
+                        },
+                    ],
+                },
+            ],
+        },
+    ];
+
+    static sendNews = async (
+        messageContext: MessageContext,
+        range: NewsRange,
+        interaction?: Eris.CommandInteraction,
+    ): Promise<void> => {
+        const locale = State.getGuildLocale(messageContext.guildID);
+
+        let summary: string;
+        try {
+            summary = State.news[range][locale];
+        } catch (err) {
+            // Failed to generate news since startup
+            logger.error(
+                `${getDebugLogHeader(messageContext)} | Error sending news: ${
+                    err.message
```

Comment:
`err` wouldnt contain any useful information would it? it would just be a null pointer reference

Reply:
That's right, I can take this log out and just print an error message